gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/ApiListItem.java

```diff
@@ -46,6 +48,10 @@ public class ApiListItem {
     @Schema(description = "Api's version. It's a simple string only used in the portal.", example = "v1.0")
     private String description;
 
+    @Schema(description = "Api's execution mode. Define if the execution mode should use v3 or jupiter mode.", example = "v3")
+    @JsonProperty(value = "execution_mode")
+    private ExecutionMode executionMode;
```

Comment:
I'm asking myself if it is necessary to add the executionMode on the `ApiListItem` entity 🧐
I'm not sure it will be a really useful information for api listing use cases. I'm also thinking about the future versions where the executionMode will certainly have no longer make sense and become really useless (and will create a breaking change if we want to remove it).

Do you think we can avoid it here ?